contracts/tests/test_deposit_locker.py

```diff
@@ -213,5 +166,190 @@ def test_event_slash(
         fromBlock=latest_block_number
     ).get_all_entries()[0]["args"]
 
-    assert event["validator"] == malicious_validator_address
+    assert event["slashedDepositor"] == malicious_validator_address
     assert event["slashedValue"] == deposit_amount
+
+
+@attr.s(auto_attribs=True)
+class Env:
+    deposit_locker: Any
+    slasher: Any
+    proxy: Any
```

Comment:
rename to `depositors_proxy`.

I tried finding a better name for `depositors` and `depositors_proxy` as one could mix them up. I could not, so please keep the full name to avoid confusion.

Reply:
Other alternative would be delegate, trustee, custodian, agent.

Reply:
proxy is being used in the context of the Env class. I fail to see any confusion arising from that.